src/platforms/dotnet/guides/winui/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: WinUI 3
+description: "Learn about using Sentry's .NET SDK with the Windows UI Library (WinUI 3)"
+redirect_from:
+  - /platforms/dotnet/winui3/
```

Comment:
How does it differ from UWP?

Their docs say:

> The Windows UI Library (WinUI) is a native user experience (UX) framework for both Windows desktop and UWP applications.

So maybe different. But below says WinUI2 as UWP and WinUI3 can be UWP or "desktop". I'm super confused with this stuff

https://docs.sentry.io/platforms/dotnet/guides/uwp/

```suggestion
  - /platform/dotnet/guides/uwp/
```